python/lsst/dax/dbserv/dbREST_v0.py

```diff
@@ -81,3 +98,48 @@ def _response(response, status_code):
     else:
         response = json.dumps(response)
     return make_response(response, status_code)
+
+
+class ColumnHelper:
+    def __init__(self, name, flags, value):
+        """
+        Helper class to define a column, get it's type for conversion, and convert types if needed
+        @param name: Column name
+        @param flags: Flags from MySQLdb.constants.FLAGS
+        @param value: An example value type to help with inferring how to convert
+        """
+        self.name = name
+        self.converter = None
+        if type(value) == float:
+            self.type = "float"
+        elif type(value) == long:
+            self.type = "long"
```

Comment:
This is not Python3-proof, `long` does not exist there. Not sure what to do about it though.